pom.xml

```diff
@@ -1727,6 +1732,7 @@
 
     <profiles>
         <profile>
+            <!-- TODO: Should this profile still exist now that we have a minimum of 17?-->
```

Comment:
IMO, the profile might be removed, and the `surefire.system.args` props set by default. As it should touch only the tests exec and we do not test even the JBoss EAP SAML adapter on JDK < 9, we should be ok.

Reply:
Agreed, I've moved this to be the default and linked the original issue in a comment for future reference.